server/routes/api/chart.py

```diff
@@ -110,6 +126,70 @@ def get_landing_page_data(dcid):
     )
     return response
 
+def get_date_to_add_for_chart(chart, sv_data):
+    """Get the date for which data is needed for a specific chart
+    
+    Args:
+        chart: the chart object that we currently care about (a single chart object from chart_config.json)
+        sv_data: the object returned from get_landing_page_data
+    
+    Returns:
+        date as a string
```

Comment:
Note that this could potentially return an empty string (and perhaps we should return None instead).